travel/res_config.py

```diff
@@ -36,6 +38,26 @@ def get_basic_passenger_limit(ir_config_parameter_pool, cr, uid, context=None):
         return limit
 
 
+def get_alert_address(ir_config_parameter_pool, cr, uid, context=None):
+    context = context or {}
```

Comment:
Please use
```
if context is None:
    context = {}
```
or the more elaborated formula that @bwrsandman suggested a time ago.